tests/unit/dataactbroker/test_file_handler.py

```diff
@@ -404,6 +404,25 @@ def test_submission_report_url_s3(monkeypatch):
     )
 
 
+def test_build_file_map_string(monkeypatch):
+    monkeypatch.setattr(fileHandler, 'CONFIG_BROKER', {'local': False})
+    upload_files = []
+    response_dict = {}
+    file_type_list = ["fabs"]
+    file_dict = {"fabs": "fabs_file.csv"}
+
+    def side_effect(filename):
+        return "123_"+filename
+    s3_url_handler = Mock()
+    s3_url_handler.get_timestamped_filename = Mock(side_effect=side_effect)
+    monkeypatch.setattr(fileHandler, 'S3Handler', s3_url_handler)
```

Comment:
Instead of this (and the previous 4 lines) you should be able to just do: ```monkeypatch.setattr(s3Handler, 'get_timestamped_filename', Mock(return_value="123+"+filename))```.

Reply:
I don't think I can (just tried)... I believe return_value doesn't have access to the calling args, so I get a NameError on 'filename'.

Reply:
Oh, I might be able to do a lambda though, lemme try that